test-data/unit/check-classes.test

```diff
@@ -3751,7 +3751,7 @@ int.__eq__(int)
 int.__eq__(3, 4)
 [builtins fixtures/args.pyi]
 [out]
-main:33: error: Too few arguments for "__eq__" of "int"
+main:33: error: Missing positional arguments 
```

Comment:
same here

Reply:
I understand callable and lamda functions. But here are you pointing at magic methods?

Reply:
If I make my message something as `Missing positional argument for call to "__eq__" of "int"` then the only difference would be language. So is it about language or the missing function name here?

Reply:
I think the original language is sufficient. The new one does not make much improvement.

Reply:
So I think where the information about the arguments to be provided is not available, i.e. the message is just "Missing positional argument", there we can leave it to the old language? Should I code it that way?

Reply:
exactly. For callables, lambdas, magic methods (and maybe more), the names of the arguments are unknown so staying to the old language is totally fine. Let's improve the language only for user-defined functions and methods in this PR. If there's a need to improve other cases in the future, we can do them later in separate PRs.